operation/periodic-task/temporal/create-namespace.sh

```
@@ -0,0 +1,65 @@
+while :; do
+  if temporal operator namespace describe -n "$NAMESPACE" --address "$TEMPORAL_ADDRESS" >/dev/null 2>&1; then
+    echo "Namespace '$NAMESPACE' already exists"
+    break
+  fi
+
+  if temporal operator namespace create -n "$NAMESPACE" --address "$TEMPORAL_ADDRESS" >/dev/null 2>&1; then
+    echo "Namespace '$NAMESPACE' created"
+    break
+  fi
+
+  if [ "$attempt" -ge "$MAX_ATTdMPTS" ]; then
```

Shellcheck (0.11.0)

[warning] MAX_ATTdMPTS is referenced but not assigned (did you mean 'MAX_ATTEMPTS'?).

(SC2154)